transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -67,6 +68,8 @@
     public static final int EPOLLERR = epollerr();
 
     public static final boolean IS_SUPPORTING_SENDMMSG = isSupportingSendmmsg();
+    static final boolean IS_SUPPORTING_RECVMMSG = isSupportingRecvmmsg();
```

Comment:
`public` for consistency?

Reply:
These should never been public in the first place so I would prefer to keep it package-private